src/Core/src/Platform/Windows/ImageViewExtensions.cs

```diff
@@ -11,9 +12,13 @@ public static void Clear(this WImage imageView)
 			imageView.Source = null;
 		}
 
-		public static void UpdateAspect(this WImage imageView, IImage image)
+		public static void UpdateAspect(this WImage imageView, IImage image, FrameworkElement element)
 		{
 			imageView.Stretch = image.Aspect.ToStretch();
+			imageView.VerticalAlignment = VerticalAlignment.Center;
+			imageView.HorizontalAlignment = HorizontalAlignment.Center;
+			element.Height = image.Height;
```

Comment:
WrapperView already handles the child control's SizeChanged event; can you set the height of the WrapperView there, and avoid having to pass the WrapperView to this method?

Reply:
It looks like WrapperView already gets its height/width from the child control: https://github.com/dotnet/maui/blob/0977c0b0c136d91e68fb0bb093c76d0f182a9abc/src/Core/src/Platform/Windows/WrapperView.cs#L84-L85

I was able to remove the binary break, but the bug isn't fixed unless I keep the map width/height stuff. Is this OK?